app/controllers/admin.js

```diff
@@ -0,0 +1,20 @@
+const services = require('../services/user');
+const log = require('../logger');
+const errors = require('../errors');
+
+exports.addAdminUser = (req, res, next) => {
+  if (!req.decode.admin) {
```

Comment:
you can add a middleware to check that the user is admin when you need this. This will permit you reuse code in others controllers